cmd/otk-resolve-containers/main_test.go

```diff
@@ -0,0 +1,194 @@
+package main_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/internal/testregistry"
+	"github.com/osbuild/images/pkg/arch"
+	"github.com/osbuild/images/pkg/blueprint"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	resolver "github.com/osbuild/images/cmd/otk-resolve-containers"
+)
+
+const rootLayer = `H4sIAAAJbogA/+SWUYqDMBCG53lP4V5g9x8dzRX2Bvtc0VIhEIhKe/wSKxgU6ktjC/O9hMzAQDL8
```

Comment:
Maybe a small comment how this is generated, what it represents etc?

Reply:
Nitpick (and fine for a followup or fine for me to take if you write a coment here in GH) - but I'm still curious where this comes from.

Reply:
Sorry.  Forgot to add this.  I'll include a comment.